guide/english/progressive-web-apps/status-bar/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Configure Status Bar
+---
+
+## Configure Status Bar
+
+In order to change the status bar at the top of the screen (which usually displays the time and battery status) of your progressive web app, you must use the configure few properties for your app to provide complete native look and feel.
+
+
+### Android
+
+Chrome, Firefox and Opera allow you to define color of the status bar using the meta tags.
+
+```html
+<!-- Chrome, Firefox OS and Opera -->
+<meta name="theme-color" content="#014473">
+```
+
+Example:
+
+<img src="https://i.imgur.com/NsUIohY.png" width="165" height="275" />
```

Comment:
Please remove these images, since they make the file heavy, these images wouldn't render in countries where imgur is blocked and if imgur removes those images, they'll not appear here